messenger.rst

```diff
@@ -1652,51 +1695,62 @@ The Redis transport DSN may looks like this:
-=======================  =====================================  =================================
-stream                   The Redis stream name                  messages
-group                    The Redis consumer group name          symfony
-consumer                 Consumer name used in Redis            consumer
-auto_setup               Create the Redis group automatically?  true
-auth                     The Redis password
-delete_after_ack         If ``true``, messages are deleted      true
-                         automatically after processing them
-delete_after_reject      If ``true``, messages are deleted      true
-                         automatically if they are rejected
-lazy                     Connect only when a connection is      false
-                         really needed
-serializer               How to serialize the final payload     ``Redis::SERIALIZER_PHP``
-                         in Redis (the
-                         ``Redis::OPT_SERIALIZER`` option)
-stream_max_entries       The maximum number of entries which    ``0`` (which means "no trimming")
-                         the stream will be trimmed to. Set
-                         it to a large enough number to
-                         avoid losing pending messages
-redeliver_timeout        Timeout before retrying a pending      ``3600``
-                         message which is owned by an
-                         abandoned consumer (if a worker died
-                         for some reason, this will occur,
-                         eventually you should retry the
-                         message) - in seconds.
-claim_interval           Interval on which pending/abandoned    ``60000`` (1 Minute)
-                         messages should be checked for to
-                         claim - in milliseconds
-persistent_id            String, if null connection is          null
-                         non-persistent.
-retry_interval           Int, value in milliseconds             ``0``
-read_timeout             Float, value in seconds                ``0``
-                         default indicates unlimited
-timeout                  Connection timeout. Float, value in    ``0``
-                         seconds default indicates unlimited
-sentinel_master          String, if null or empty Sentinel      null
-redis_sentinel           support is disabled
-ssl                      Map of TLS options.                    null
-=======================  =====================================  =================================
-
-The ``ssl`` option can be used to change requirements for the TLS channel, e.g. in tests:
+``stream`` (default: ``messages``)
+    The Redis stream name
 
-.. configuration-block::
+``group`` (default: ``symfony``)
+    The Redis consumer group name
+
+``consumer`` (default: ``consumer``)
+    Consumer name used in Redis
+
+``auto_setup`` (default: ``true``)
+    Create the Redis group automatically?
```

Comment:
I find it strange to have a question here, what do you think?

Reply:
You are right. I renamed it as `Whether to create the Redis group automatically`. Thanks.